google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2498,6 +2498,19 @@ Blob create(
    * <p>Note that the {@link ServiceAccountSigner} may require additional configuration to enable
    * URL signing. See the documentation for the implementation for more details.
    *
+   * <p>Example of creating a signed URL for a blob with generation.
+   *
+   * <pre>{@code
+   * String bucketName = "my-unique-bucket";
+   * String blobName = "my-blob-name";
+   * long generation = 1576656755290328L;
+   *
+   * URL signedUrl = storage.signUrl(
+   *     BlobInfo.newBuilder(bucketName, blobName, generation).build(),
+   *     7, TimeUnit.DAYS,
+   *     SignUrlOption.withQueryParams(ImmutableMap.of("generation", "" + generation)));
```

Comment:
`SignUrlOption.withQueryParams(ImmutableMap.of("generation", "" + generation)));` => ` SignUrlOption.withQueryParams(ImmutableMap.of("generation",  generation)));`

Reply:
it won't compile, the type of 'generation' is long it should be coverted to String

Reply:
Ahh, My bad I assume it to be string without checking.

Still, I think `SignUrlOption.withQueryParams(ImmutableMap.of("generation", String.valueOf(generation))));` is more readable

Reply:
okay, fixed